packages/plugins/ganache/src/index.js

```diff
@@ -1,10 +1,41 @@
+const constants = require('embark-core/constants');
+
 class Ganache {
   constructor(embark) {
-    embark.events.request('blockchain:vm:register', () => {
+    embark.events.request("blockchain:node:register", constants.blockchain.clients.ganache, {
+      isStartedFn: (cb) => {
+        cb(null, true); // Always assume it's started, because it's just a provider (nothing to start)
+      },
+      launchFn: (cb) => {
```

Comment:
I do believe this gets called every time embark starts by way of `blockchain:node:start`.

Reply:
as the comment says this is fine still since the callback will work. it seems the callback is obligatory though. we could make some of this fields optional thought to make it easier read (but different PR). ie so launchFn: is undefined has default `(cb) => {cb()}}`

Reply:
The comments says `// Will probably never be called, but if so, callback straight aways because there is nothing to start`

So my suggestion would be to remove the `will probably never but called, but if so,` part, as it is misleading.

Reply:
ah ok I missunderstood, I thought it was pointing out it would be an issue

Reply:
I don't think `launchFn` is called, because how the blockchain-client works is it checks if the node is already started, if it **is**, then it doesn't call start. See above, I always return `true` to `isStartedFn`